flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskManagerServices.java

```diff
@@ -145,6 +151,10 @@ public TaskEventDispatcher getTaskEventDispatcher() {
 		return taskEventDispatcher;
 	}
 
+	public ExecutorService getIOExecutor() {
```

Comment:
```suggestion
	public Executor getIOExecutor() {
```

It's enough to return `Executor` because we do not need lifecycle management methods from such as `shutdown()` from outside.